harvard-swinburne-university-of-technology.csl

```diff
@@ -167,7 +186,11 @@
           <text macro="author"/>
           <text macro="year-date"/>
         </group>
-        <text macro="title"/>
+        <choose>
+          <if match="any" variable="author editor translator">
```

Comment:
Not sure I understand the need for this if here? If the title is used instead of author via substitute, it gets suppressed here anyway

Reply:
Oh does it? Sorry, I didn't realise it behaved like that. I'll fix it up